### PR TITLE
Bugfix for oddly-oriented files

### PR DESCRIPTION
**Pull request by keflavich**
The Dame et al 2001 CO cube is oriented differently from most cubes and currently fails to load with an error like:

```
  File "<ipython-input-2-42b6621b6051>", line 1, in <module>
    cube = SpectralCube.read('/Users/adam/Downloads/new_DHT02_Center_bw_mom.fits')
  File "/Users/adam/repos/spectral-cube/spectral_cube/spectral_cube.py", line 1956, in read
    cube = read(filename, format=format, hdu=hdu, **kwargs)
  File "/Users/adam/repos/spectral-cube/spectral_cube/io/core.py", line 33, in read
    return load_fits_cube(filename, hdu=hdu, **kwargs)
  File "/Users/adam/repos/spectral-cube/spectral_cube/io/fits.py", line 158, in load_fits_cube
    cube = SpectralCube(data, wcs, mask, meta=meta, header=header)
  File "/Users/adam/repos/spectral-cube/spectral_cube/spectral_cube.py", line 2353, in __init__
    beam = cube_utils.try_load_beam(self.header)
  File "/Users/adam/repos/spectral-cube/spectral_cube/spectral_cube.py", line 2171, in header
    header['CDELT3'] *= self._spectral_scale
TypeError: unsupported operand type(s) for *=: 'float' and 'NoneType'
```

This is occurring because `self._spectral_scale` is not being set, which in turn happens because `CUNIT3` is not in the header.

We missed this in tests because the rescale-spectral-unit functionality is never invoked for our test cubes, which have an *assumed* m/s unit.  So I've updated the test data to be in km/s (and explicitly set CUNIT in the test headers) to force this to be tested, which may break some more things we'll have to fix....

### CODE REVIEW

**Comment by keflavich** [comment]
The build failures appear to be related to astropy-helpers.  @bsipocz, any tips here?

**Comment by bsipocz** [comment]
Hmm, that error is very weird. Could you ping me to come back to it next week, I'll be offline for a few days.

File: .travis.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ env:
         - SETUP_CMD='test -a "--boxed"'
         - NUMPY_VERSION=1.13
         - ASTROPY_VERSION=stable
-        - CONDA_DEPENDENCIES='matplotlib aplpy pytest pytest-xdist'
+        - CONDA_DEPENDENCIES='matplotlib aplpy pytest pytest-xdist astropy-helpers'
```
Comment (bsipocz) [inline .travis.yml]: I wouldn't do this, it should be possible to sort this out having the helpers as a submodule rather than as a hard dependency

**Comment by keflavich** [comment]
Thanks @bsipocz, but it's a necessary workaround right now.  I'll happily replace the dumb hacks when we have an alternative.

**Pull request merged**